server/modules/socket_endpoints/ScreenSocket.js

```diff
@@ -49,3 +60,19 @@ eventBus.on("player-update", () => {
 eventBus.on("new-video-timestamp", (timestamp) => {
   endpoint.emit("new-video-timestamp", timestamp);
 });
+
+function emitNewPhoto() {
```

Comment:
I think its nicer to give a photo its own socket and give that an interval.
You could also let the frontend fetch the photos as there is no api key needed for those photos.

Reply:
Disagree, I only fetch a small JSONobject on the server with the link of the image so all the screens show the same photo. End I think the queue update is the best place as this one is only shared with the screen and then we would need to make a more complicated system to guard that a photo update is sent when a queue update is not and vice-versa, as it should be one or the other. Therefore I think sending it with an empty queue update makes more sense.